go.mod

```diff
@@ -84,14 +84,15 @@ replace (
 	github.com/DataDog/datadog-agent/pkg/util/testutil => ./pkg/util/testutil
 	github.com/DataDog/datadog-agent/pkg/util/winutil => ./pkg/util/winutil/
 	github.com/DataDog/datadog-agent/pkg/version => ./pkg/version
+	gopkg.in/DataDog/dd-trace-go.v1 => /Users/gustavo.caso/go/src/github.com/DataDog/dd-trace-go
```

Comment:
I needed this to test my local copy

Reply:
@GustavoCaso How does the output of `go tool nm $serverless_binary | grep go-tuf` look like?

Also, are the only required changes to support this moving the interfaces `Propagator`, `TextMapReader` and `TextMapWriter` to a separate package? If yes, then we have a solution for Serverless.

The only difference that I foresee in the v2 implementation is that these interfaces won't be under `ddtrace` package, but a `propagation` package inside of the `tracer` package. As far as the `propagation` package doesn't import anything from `tracer` itself, it won't include any extra dependencies.